config/helper/mailer.php

```
@@ -0,0 +1,66 @@
+    $mail = new PHPMailer(true);
+
+    try {
+        $mail->isSMTP();
+        $mail->Host = 'smtp.gmail.com';
+        $mail->SMTPAuth = true;
+        $mail->Username = 'feizyaira@gmail.com';
+        $mail->Password = 'pgtpnwpudkzyftoo';
+        $mail->SMTPSecure = 'tls';
+        $mail->Port = 587;
+
+        $mail->setFrom($env['MAILER_USERNAME'] ?? getenv('MAILER_USERNAME'), 'InviezID Admin');
```

PHPMD (2.15.0)

Avoid unused local variables such as '$env'. (Unused Code Rules)

(UnusedLocalVariable)